src/main/java/com/github/alina/sesion4_challenge/ChallengeEighteen.java

```diff
@@ -0,0 +1,24 @@
+package com.github.alina.sesion4_challenge;
+//Write a Java program where you declare an integer array with 5 elements. Use a for loop to initialize the array such that each element is equal to its index incremented by 1. Then use another for loop to decrement each element by 1. Print the array before and after the decrement operation using a for-each loop
+public class ChallengeEighteen {
+    public static void main(String[] args) {
+        int[] array = new int[5];
+
+        // Initialize the array elements using a for loop
+        for (int i = 0; i < array.length; i++) {
```

Comment:
We call variables like `i`, `magic numbers` because they lack description (and they are `magic` because we don't know what they are doing). When we solve some algorithmic exercises, we can use variables like `i`, but often, a good practice would be to have a more comprehensive variable description (eg `index`, in our case).

Reply:
Replace i with index and also add private static final var for array size.